core-web/libs/portlets/dot-ema/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -436,6 +441,7 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
                         this.resetDialogIframeData();
                         this.reloadIframe();
                         this.savePayload = undefined;
+                        this.cd.detectChanges();
```

Comment:
I'm not sure because I have not tested that. But we can know for a fact that this component is not emitting anything, so maybe all our change detection problems can be solved by just deleting the L88 in this file because this component will never push anything to the parent nor receive inputs therefore the OnPush will not detect changes when we need it unless we do this. We can give it a shot an see how it behaves.

cc: @fmontes

Reply:
I think we are having this issues because the events are coming from the iframe we might want to use ngzone to not have to do `detectChanges`